### PR TITLE
hungarian localisation

### PR DESCRIPTION
**Pull request by lesorubshayan**

### CODE REVIEW

File: runtime/lang/menu_hu_hu.utf-8.vim [hunk 1]
```diff
@@ -246,3 +246,29 @@ let g:menutrans_fileformat_dialog = "Válaszd ki a fájl formátumát"
...
+menutrans Set\ language\ to\ "en"				Nyelv\ beállítása\ en
+menutrans Set\ language\ to\ "en_au"				Nyelv\ beállítása\ en_au
+menutrans Set\ language\ to\ "en_ca"				Nyelv\ beállítása\ en_ca
+menutrans Set\ language\ to\ "en_gb"				Nyelv\ beállítása\ en_gb
+menutrans Set\ language\ to\ "en_nz"				Nyelv\ beállítása\ en_nz
+menutrans Set\ language\ to\ "en_us"				Nyelv\ beállítása\ en_us
+menutrans &Find\ More\ Languages				További\ nyelvek\ keresése
+let g:menutrans_set_lang_to =					'Nyelv beállítása en_us'
+menutrans &Show\ File\ Types\ in\ menu				Fájltípusok\ megjelenítése\ a\ menüben
+menutrans &Sponsor/Register					Szponzor/Regisztráció
```
Comment (chrisbra) [inline runtime/lang/menu_hu_hu.utf-8.vim:271]: Same here, please move above the `let &cpo` line

**Comment by ronin49** [comment]
@chrisbra

**Comment by chrisbra** [comment]
ping @arpadffy

**Comment by chrisbra** [comment]
closing for the reasons mentioned here https://github.com/vim/vim/pull/14703#issuecomment-2092337585

**Pull request closed without merging**